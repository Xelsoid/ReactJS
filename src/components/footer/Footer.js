import React from 'react';

import './footer.scss';

const Footer = () => (
  <footer className="footer">
    <span>&copy; All rights reserved</span>
  </footer>
);

export default Footer;
