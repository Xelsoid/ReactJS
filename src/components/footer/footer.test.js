import React from 'react';
import Footer from './Footer';

it('renders correctly', () => {
    const component = shallow (
      <Footer />
    );
    expect(component).toMatchSnapshot();
});

