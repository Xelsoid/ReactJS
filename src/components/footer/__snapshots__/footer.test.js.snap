// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="footer"
>
  <span>
    Â© All rights reserved
  </span>
</div>
`;
